### PR TITLE
[bors] add bors integration

### PR DESCRIPTION
**Pull request by Saviq**

### CODE REVIEW

**Comment by Saviq** [comment]
bors try

File: bors.toml [hunk 1]
```diff
@@ -0,0 +1 @@
+status = ["Snap"]
```
Comment (AlanGriffiths) [inline bors.toml:1]: Can we have:
```plain
delete-merged-branches = true
```

**Review by AlanGriffiths — APPROVED** [review]
bor r+

**Comment by Saviq** [comment]
bors r=AlanGriffiths

**Pull request merged**